Source/Core/Core/Lua/LuaFunctions/LuaEmuFunctions.cpp

```diff
@@ -0,0 +1,137 @@
+#include "LuaEmuFunctions.h"
+#include "../LuaVersionResolver.h"
+#include "Core/State.h"
+#include "Core/Core.h"
+#include "Core/Movie.h"
+#include "Core/PowerPC/PowerPC.h"
+#include <optional>
+
+namespace Lua
+{
+namespace LuaEmu
+{
+  bool waitingForSaveStateLoad = false;
```

Comment:
likewise these should probably all also be static. Even better, these should be put into the instance `emu` class below, so their constructors (in particular the strings) are only run when the actual instance is allocated, rather than at program startup.

Ditto for similar other lua modules like this